modules/exploits/unix/webapp/wp_revslider_upload_execute.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::HTTP::Wordpress
+  include Msf::Exploit::FileDropper
+  require 'rex/zip'
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Wordpress RevSlider File Upload and Execute Vulnerability',
+      'Description'    => %q{
+        This module exploits an arbitrary PHP code upload in the WordPress ThemePunch
+        Revolution Slider ( revslider ) plugin, version 3.0.95 and prior. The
+        vulnerability allows for arbitrary file upload and remote code execution.
+      },
+      'Author'         =>
+        [
+          'Simo Ben youssef', # Vulnerability discovery
+          'Tom Sellers <tom[at]fadedcode.net>'  # Metasploit module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['URL', 'https://whatisgon.wordpress.com/2014/11/30/another-revslider-vulnerability/'],
+          ['URL', 'https://www.exploit-db.com/exploits/35385/'],
+          ['EDB', '35385'],
+          ['OSVDB', '115118']
```

Comment:
It's not the same vuln.  WPVDB has two vulnerabilities conflated. That one is a arbitrary file download that exists in 4.1.4 and earlier.